### PR TITLE
Namespace function as script file 

### PR DESCRIPTION
**Pull request by devel-pa**
and included with Handlebars runtime.

### CODE REVIEW

**Comment by es128** [comment]
I assume you tested this out, including all the scenarios you had trouble with before?

File: test.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 var expect = require('chai').expect;
 var Plugin = require('./');
 var Handlebars = require('handlebars');
+var sysPath = require('path');
+var sysFile = require('fs');
+ 
+var nsPath = sysPath.join(__dirname, 'ns.js'); 
+var nsFile = sysFile.readFileSync(nsPath, {encoding : 'utf8' });
+eval(nsFile);
```
Comment (es128) [inline test.js]: Hmm, I guess there's no better way to do this, is there?

Please add some comments explaining what's happening here and why it's being done like this.

Comment (es128) [inline test.js]: Actually, I'm not sure I get why `require('./ns')` wouldn't work.

**Comment by devel-pa** [comment]
Yes, tested all scenarios I'm aware of. Also test file updated.

**Comment by devel-pa** [comment]
`ns.js` file is not a node module and must be executed in Handlebars context.

**Comment by es128** [comment]
have you tried it? If you do something like

```js
globals.Handlebars = Handlebars;
require('./ns');
```

it might work

**Comment by devel-pa** [comment]
You're right, I didn't thought about doing that. (with `global`)

**Comment by devel-pa** [comment]
Thank you

**Comment by es128** [comment]
Thanks for contributing back to the project. I hope others will find these new features useful as well.

**Pull request merged**